README.md

```diff
@@ -49,6 +49,15 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### Linux
+
+You may need to set up access control for the X server for this to work. Install the xhost package (one is available for all distros) and run:
+
+```sh
+export DISPLAY=:0.0
```

Comment:
Please try this and see if it works for you:
```suggestion
export DISPLAY=:0
```

I think you only need the `:0.0` with multiple screens, https://unix.stackexchange.com/a/16816/53208

Reply:
Confirming that `export DISPLAY=:0` works for WSL Ubuntu on Win11.

Reply:
I can confirm that `export DISPLAY=:0` works on Linux **with X11** and a multiscreen environment.
It does **not** work with **Wayland**